inst/EpiModel2.rmd

```diff
@@ -112,6 +120,7 @@ The package `tergmLite` has been added as a dependent package for EpiModel. `ter
 The use of `tergmLite` is fairly straightforward and uses one of two versions of the full network object during network simulations. If `tergmLite = TRUE` is passed in as a control setting, EpiModel will use the `networkLite` version of the full network, a two-column edgelist representation of the full network. Alternatively, if `tergmLite = FALSE` is passed through, EpiModel will use the original `network` object for resimulations. As an example:
```

Comment:
suggested changes to line 104: copy-edits

>  EpiModel aims to assist new users in their modeling efforts while being flexible enough to allow for complex simulation scenarios not handled by base EpiModel modules. If a user passes in an epidemic type through the `control.net` input parameter `type` (e.g., `type = SI`, `SIR`, or `SIS`) EpiModel will run the model as specified, using input parameters to automate the selection of appropriate base modules. If, however, the user does not pass in a specified epidemic type, or passes in `type = NULL` to control settings, then EpiModel will run the model as specified by the user; this includes running any non-standard base modules or additional user-created modules in their workflow (examples of which are provided in the "Worked Example" section of this document). Conversely, if a user passes any non-standard base modules or module extensions, the epidemic type is required to be `NULL`. EpiModel will return an error if these rules are not followed. This change was made to emphasize to more advanced users of EpiModel (e.g., in the development module extensions) that EpiModel will run the model as specified, though the results may not be as expected.

suggested changes to line 106: "built in" -> "built-in"

Is "Two-Group Functions" referring to a specific section? It seems like that section has been renamed. Maybe change to be more clear of what specific section you're referring to.

> Note: for two-group models, if `type` is `NULL`, the user must be mindful to select the correct version of built-in modules as described in Two-Group Functions above.

suggested changes to line 110: copy-edits

> Another important change made to the EpiModel workflow was implemented in EpiModel 1.70 but bears repeating: moving away from the language of "births" and "deaths" in models with vital dynamics to "arrivals" and "departures". This change accounts for a wider range of models in which entry into and exiting from the population is not necessarily through a mortality process (e.g., entry and exit through an aging process). To this end, all input parameters based on births and deaths have been renamed based on arrivals and departures. For example the birth rate `b.rate` passed to `param.net`, has been renamed `a.rate` for "arrival rate." The rate for deaths remains named `d.rate` though its interpretation has changed to that of departures. Module functions based on birth and death rates have been changed: `births.FUN` and its associated base module `births.net` are now `arrivals.FUN` and `arrivals.net` (`arrivals.2g.net` in the two group case) respectively. Similar changes exist for `deaths.FUN` and `deaths.net`, which are now `departures.FUN` and `departures.net` respectively.

suggested changes to line 114: copy-edits

> Along with the changes to EpiModel detailed in the previous section, several additions have been made to EpiModel and the EpiModel workflow to improve usability and to clarify the overall epidemic process simulated by EpiModel.

suggested changes to line 118: copy-edits

> The package `tergmLite` has been added as a dependent package for EpiModel. `tergmLite` reduces the overall computational burden that comes with using the full network object in EpiModel simulations. For users whose research needs require only summary epidemiological statistics, `tergmLite` gives significant computational efficiency, with simulation times sped up between 25- to 50-fold depending on the model in question. Details of `tergmLite`, as well as examples of what types of research projects it may be useful for, can be found on Github at [tergmLite](https://github.com/statnet/tergmLite).

suggested changes to line 120:

```suggestion
`tergmLite` uses one of two versions of the full network object during network simulations. If `tergmLite = TRUE` is passed in as a control setting, EpiModel will use the `networkLite` version of the full network, a two-column edgelist representation of the full network. Alternatively, if `tergmLite = FALSE` is passed through, EpiModel will use the original `network` object for resimulations. As an example:
```